<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="DataDrivenFramework.Login">
        <methods>
          <include name="loginWithBothCorrect" invocation-numbers="0 1 2 3"/>
        </methods>
      </class> <!-- DataDrivenFramework.Login -->
      <class name="loginPackage.DependenciesManagement">
        <methods>
          <include name="oneTest"/>
        </methods>
      </class> <!-- loginPackage.DependenciesManagement -->
      <class name="DataDrivenFramework.LoginFromJXL">
        <methods>
          <include name="loginWithBothCorrect" invocation-numbers="0 1 2 3"/>
          <include name="launchBrowser"/>
          <include name="closeBrowser"/>
        </methods>
      </class> <!-- DataDrivenFramework.LoginFromJXL -->
      <class name="InterviewQuestions.RetryFailedTest">
        <methods>
          <include name="testCase1"/>
          <include name="testCase2"/>
        </methods>
      </class> <!-- InterviewQuestions.RetryFailedTest -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
